egret/viz/generate_graphs.py

```diff
@@ -390,9 +396,9 @@ def _plot_generation_stack_components():
             reserves_by_hour += reserves_available
     
     if reserve_requirements_by_hour is not None:
-        implicit_reserves_by_hour = [max(0.0, reserves_by_hour[ix] - reserves_by_hour[ix]) for ix in range(len(reserve_requirements_by_hour))]
+        implicit_reserves_by_hour = np.maximum(0.0, reserves_by_hour - reserve_requirements_by_hour)
```

Comment:
I think `reserve_requirements_by_hour` needs the `attribute_to_array` treatment. (See TravisCI build error (subtraction between float and dictionary))